remove hardcoded /usr/include for build fastmath component of pycrypto;
it causes problems in a sysroot build environment, and it's not actually needed anyway
since /usr/include is a standard location
author: Kenneth Hoste (HPC-UGent)
--- pycrypto-2.6.1/setup.py.orig	2020-09-01 10:04:38.890764478 +0200
+++ pycrypto-2.6.1/setup.py	2020-09-01 10:04:51.061886099 +0200
@@ -370,7 +370,7 @@
       'ext_modules': plat_ext + [
             # _fastmath (uses GNU mp library)
             Extension("Crypto.PublicKey._fastmath",
-                      include_dirs=['src/','/usr/include/'],
+                      include_dirs=['src/'],
                       libraries=['gmp'],
                       sources=["src/_fastmath.c"]),
 
